Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM nginx:latest
+
+# Install fundamental packages
+RUN apt update
+RUN apt install -y\
+	sqlite3 \
+	gettext \
```

Comment:
Not needed, we're using a PHP implementation of gettext asd

Reply:
I mean just the line that says "gettext", GitHub shows a 4 lines snippet for no reason

Reply:
@lvps It is for the `msgfmt` command in the setup process, should I remove it?